Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportEntitiesDataMapping.cs

```diff
@@ -18,6 +18,9 @@ namespace Amdocs.Ginger.CoreNET.Run.RunListenerLib.CenteralizedExecutionLogger
 {
     public static class AccountReportEntitiesDataMapping 
     {     
+             //select template 
+        static HTMLReportConfiguration _HTMLReportConfig = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<HTMLReportConfiguration>().Where(x => x.IsDefault ).FirstOrDefault();
+           
         public static AccountReportAction MapActionStartData(GingerCore.Actions.Act action, Context context)
```

Comment:
this is ActionStart or ActivityStart? methos name says ActionStart but inside it looks like ActivityStart